tools/cli_install.py

```diff
@@ -144,21 +158,32 @@ def usage(f=sys.stderr):
         f.write("usage: cli_install.py <path_to_existing_cli> <target_dir>\n")
         f.write("  OR\n")
         f.write("usage: cli_install.py <cluster_url> <target_dir>\n")
+        f.write("  OR\n")
+        f.write("usage: cli_install.py ensure_installed <cluster_url> <target_dir>\n")
 
 
     if not len(sys.argv) == 3:
         usage()
         sys.exit(1)
 
-    source = sys.argv[1]
-    target_dir = sys.argv[2]
+    args = sys.argv[1:]
+    maybe_install = False
+    if args[0] == "ensure_installed":
+        maybe_install=True
+        args=args[1:]
+
+    source = args[1]
+    target_dir = args[2]
 
     # convenience for command line
     if not os.path.exists(target_dir):
         os.makedirs(target_dir)
 
     if source.startswith('http://') or source.startswith('https://'):
-        download_cli(source, target_dir)
+        if maybe_install:
```

Comment:
Correct me if I'm wrong, but download_cli will overwrite an already installed CLI, correct?

Will we actually ever want to do that?